templates/front_page.html

```diff
@@ -18,6 +18,7 @@
 
 
 {% block content %}
+<body background="https://images.pexels.com/photos/259915/pexels-photo-259915.jpeg?auto=compress&cs=tinysrgb&h=350">
```

Comment:
The file is a [Jinja template](http://jinja.pocoo.org/), not a regular HTML file.
Consider this: the **front_page.html** indirectly (through public.html or logged.html) _extends_ **main.html**. So the `<body>` tag is defined in **main.html** and it sort of gets imported in **front_page.html** file.

So the body tag that you've put here is invalid as it's not the right place for it.